src/components/ClusterManager.js

```diff
@@ -310,16 +293,16 @@ export default class ClusterManager extends PureComponent {
           tooltip: Utils.cond(
             [!canCompute, () => noCompute],
             [creating, () => 'Your environment is being created'],
-            () => 'Update runtime'
+            () => 'Update cloud environment'
           ),
           onClick: () => this.setState({ createModalDrawerOpen: true }),
           disabled: isDisabled
         }, [
           div({ style: { marginLeft: '0.5rem', paddingRight: '0.5rem', color: colors.dark() } }, [
-            div({ style: { fontSize: 12, fontWeight: 'bold' } }, 'Notebook Runtime'),
+            div({ style: { fontSize: 12, fontWeight: 'bold' } }, 'Cloud Environment'),
             div({ style: { fontSize: 10 } }, [
               span({ style: { textTransform: 'uppercase', fontWeight: 500 } }, [currentStatus === 'LeoReconfiguring' ? 'Updating' : (currentStatus || 'None')]),
-              currentStatus && ` (${Utils.formatUSD(totalCost)} hr)`
+              !!totalCost && ` (${Utils.formatUSD(totalCost)} hr)`
```

Comment:
Why do we not still need currentStatus here? I would expect we might still want to know if there is a runtime or a disk before displaying this at all.

Reply:
currentStatus I think was used to determine if there was an existing cloud environment previously. Now totalCost (which is the sum of existing disk and cloud vm) will let us know if we have a current cloud enviornment since if not it would be 0 which i think is falsy

Reply:
Right, and `totalCost` could be non-zero without a runtime if there is a floating persistent disk.